corehq/apps/locations/models.py

```diff
@@ -94,20 +94,20 @@ class LocationType(models.Model):
         null=True,
         related_name='+',
         db_column='expand_from',
-        on_delete=models.PROTECT,
+        on_delete=models.RESTRICT,
```

Comment:
I assume it's minimal, but it seems like `RESTRICT` could impose extra query overhead and thus affect delete performance. Has that been considered?